blog/20130402mate-1-6-released.md

```diff
@@ -151,7 +151,7 @@ With MATE 1.6 I can feel right back at home on my favorite Linux distribution
 and get back to work.
 
 We also asked for a comment from the
-[company who donated the server where we host MATE services](/blog/2012-12-27-thank-you-first-colo/). **Martin Verges**, CEO of [FirstColo GmbH](http://www.first-colo.net/EN/index.php):
+[company who donated the server where we host MATE services](/blog/2012-12-27-thank-you-first-colo/). **Martin Verges**, CEO of [FirstColo GmbH](https://www.first-colo.net/EN/index.php):
```

Comment:
https://www.first-colo.net/EN/index.php shows error 404

I think you can use https://www.first-colo.net/en/ it works